java/impl/aws/src/main/java/co/worklytics/psoxy/S3Handler.java

```diff
@@ -50,21 +62,37 @@ public String handleRequest(S3Event s3Event, Context context) {
 
         log.info(String.format("Received a request for processing %s from bucket %s.", sourceKey, importBucket));
 
-        S3Object s3Object = s3Client.getObject(new GetObjectRequest(importBucket, sourceKey));
+
+        List<StorageHandler.ObjectTransform> transforms = Lists.newArrayList(
+            StorageHandler.ObjectTransform.of(destinationBucket,  (CsvRules) defaultRules));
+
+        rulesUtils.parseAdditionalTransforms(configService)
+            .forEach(transforms::add);
+
+        for (StorageHandler.ObjectTransform transform : transforms) {
+            process(importBucket, sourceKey, transform.getDestinationBucketName(), transform.getRules());
+        }
+
+        return "Processed!";
```

Comment:
"200-OK", not sure where this is read back (probably nowhere) but if so, easier to process